templates/vue/src/components/Lightbox/media/AnswerMedia.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="answers">
+    <h3>{{ node.title }}</h3>
+    <div
+      v-if="answer !== null"
+      class="answer-container mx-auto mb-3"
+      data-qa="answer-display"
+    >
+      <h4>{{ answer.precedingText || question.text }}</h4>
+      <b-tabs vertical no-nav-style nav-class="nav-tablist">
+        <b-tab v-for="questionAnswer in answers" :key="questionAnswer.type">
+          <template #title>
+            <div class="icon">
+              <tapestry-icon :icon="questionAnswer[0]" />
+            </div>
+          </template>
+          <tapestry-activity
+            v-if="questionAnswer[0] === 'audio'"
+            :type="questionAnswer[0]"
+            :answerData="getFullUrl(questionAnswer[1].url)"
+          ></tapestry-activity>
+          <tapestry-activity
+            v-else
+            :type="questionAnswer[0]"
+            :answerData="questionAnswer[1]"
+          ></tapestry-activity>
+        </b-tab>
+      </b-tabs>
+      <div v-show="!hasAnswer" class="media-wrapper">
+        You have not completed this question yet.
+      </div>
+    </div>
+    <div v-else>
+      <p>Please fill in the answer form.</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapState } from "vuex"
+import TapestryActivity from "./ActivityMedia/TapestryActivity"
+import TapestryIcon from "@/components/common/TapestryIcon"
+import { data as wpData } from "@/services/wp"
+
+export default {
+  name: "answer-media",
+  components: {
+    TapestryActivity,
+    TapestryIcon,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    ...mapState(["userAnswers"]),
+    ...mapGetters(["getQuestion", "getAnswers"]),
+    answer() {
+      return this.node.typeData

```

Comment:
This is a bit misleading because answer != typeData really. Rather than use this computed property, I think it would make more sense to use the actual thing you are looking for, e.g. `this.node.typeDada.questionId`, etc. Remember that typeData could potentially contain other fields that do not necessarily mean this node is an answer node.

Reply:
This seems like a computed component used in order to not access this.node.typdeData all the time. I changed the name for this property for clarity.